src/averageAges.js

```diff
@@ -14,12 +14,27 @@
  *
  * @return {number}
  */
+const getAverageAge = (people) => {
+  const sumOfAge = people.reduce((sum, person) => {
+    const age = person.died - person.born;
+
+    return sum + age;
+  }, 0);
+
+  const averageAge = sumOfAge / people.length;
+
+  return Math.round(averageAge * 100) / 100;
```

Comment:
Instead of manually rounding the average age to two decimal places, consider using the toFixed method. E.g., averageAge.toFixed(2).